Assets/SEE/Utils/ColorExtensions.cs

```diff
@@ -81,7 +81,7 @@ public static Color Invert(this Color color)
         /// <returns>The converted colors as gradient color keys.</returns>
         public static IEnumerable<GradientColorKey> ToGradientColorKeys(this ICollection<Color> colors)
         {
-            return colors.Select((c, i) => new GradientColorKey(c, (float)i / colors.Count));
+            return colors.Select((c, i) => new GradientColorKey(c, (float) i / (colors.Count-1)));
```

Comment:
Is there a chance of divivion by zero?

Reply:
Yes, there is, good catch. Fixed it now.